### PR TITLE
bitbucket PG: fix default livecheck

### PR DESCRIPTION
**Pull request by l2dy**
for ports using bitbucket.tarball_from downloads

### CODE REVIEW

**Comment by seanfarley** [comment]
I need to check this more thoroughly but it might be ok. I'm unsure it works with some of the branch parsing I do in a few ports.

File: _resources/port1.0/group/bitbucket-1.0.tcl [hunk 1]
```diff
@@ -57,6 +57,7 @@ proc handle_tarball_from {option action args} {
     if {[string equal ${action} "set"] && ${args} eq "downloads"} {
         bitbucket.tarball_from ${args}
         bitbucket.master_sites https://bitbucket.org/${bitbucket.author}/${bitbucket.project}/downloads
+        default livecheck.url https://bitbucket.org/${bitbucket.author}/${bitbucket.project}/downloads
```
Comment (seanfarley) [inline _resources/port1.0/group/bitbucket-1.0.tcl]: Why not just:

```tcl
default livecheck.url ${bitbucket.master_sites}
```

?

**Comment by seanfarley** [comment]
Besides my one nit, this should be fine.

**Comment by l2dy** [comment]
@seanfarley Fixed.

**Comment by seanfarley** [comment]
Rebased and pushed. Thanks!

**Pull request closed without merging**